sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -5106,7 +5106,7 @@ object SQLConf {
         "the sequence of steps that the query performs in a composable fashion.")
       .version("4.0.0")
       .booleanConf
-      .createWithDefault(Utils.isTesting)
+      .createWithDefault(true)
```

Comment:
Is this a mistake? This should be a part of this PR, @dtenedor .

Reply:
@dongjoon-hyun Good question; this is intentional, the feature is done as of this PR and ready to be enabled by default. There are no backwards compatibility issues as the new syntax will work with traditional SQL queries with no breaking changes.

Let me know if you prefer to split this config into a separate PR given this information and I can do that.